packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeErrorRenderer.tsx

```diff
@@ -37,16 +37,30 @@ export const TreeErrorRenderer: React.ForwardRefExoticComponent<TreeErrorRendere
                 }
               : undefined
           }
+          aria-level={0}
```

Comment:
I think error nodes should use information from 'flattened node'. `aria-level={0}` does not look like correct approach. Also why only `NoFilterMatches` node has `aria-posinset` and `aria-setsize` set as 0?

Reply:
Changed to use level from flattened node, missed `NoFilterMatches`, changed set size and pos in set to 1

Reply:
Why some type of nodes still have `setsize` and `posinset` values set to constants instead of using information from `FlatNode`?

Reply:
I don't think there is a reason to clutter interfaces with `setsize` and `posinset` when we know that their pos and set size is a constant 1.

Reply:
Can't we just pass whole `FlatNode` to those places?

Reply:
We already check if its an info (error node), so we'd need to add flat properties inside it.

Reply:
You could do something like:

```ts
type FlatTreeNode<TNode extends PresentationTreeNode = PresentationTreeNode> = {
  level: number;
  levelSize: number;
  posInLevel: number;
} & TNode;

interface TreeErrorRendererOwnProps {
  node: FlatTreeNode<PresentationInfoNode>;
}
```

Reply:
Changed Flat tree node interface to the one suggested, seems more flexible.